cmd/admin-handlers-users.go

```diff
@@ -526,7 +530,7 @@ func (a adminAPIHandlers) AddServiceAccount(w http.ResponseWriter, r *http.Reque
 
 	// Need permission if we are creating a service acccount
 	// for a user <> to the request sender
-	if targetUser != "" && targetUser != cred.AccessKey {
+	if targetUser != cred.AccessKey {
```

Comment:
@donatello I think this is a breaking change, an account can create a service account for itself when target user is empty. Could you check with @Alevsk to check if console will break or not after this change ?

Reply:
Yes that is correct and a good catch - targetUser is allowed to be empty to inherit the same parentUser as incoming user @donatello - I assume you tested this with `console UI` as well.

Reply:
Will check - missed it earlier. We could just have targetUser set to the incoming user, instead of handling an empty targetUser case, which I removed it in this change.

Reply:
Fixed in latest change.